GOVERNANCE.md

```diff
@@ -15,6 +15,9 @@ Current core team members are:
 | Geraldine Van der Auwera | Broad Institute | [vdauwera](https://github.com/vdauwera) |
 | Christopher Llanwarne | Broad Institute | [cjllanwarne](https://github.com/cjllanwarne) |
 | John Didion | DNAnexus | [jdidion](https://github.com/jdidion) |
+| Michael Franklin | Australian Centre for Population Genomics | [illusional](https://github.com/illusional) |
```

Comment:
Hey @patmagee! I phrased this poorly in the slack message, hoping this could be straight _Centre for Population Genomics_.

```suggestion
| Michael Franklin | Centre for Population Genomics | [illusional](https://github.com/illusional) |
```